Assets/PatchKit Patcher/Scripts/AppData/Remote/RemoteMetaData.cs

```diff
@@ -75,5 +83,12 @@ public string GetKeySecret(string key, string cachedKeySecret)
 
             return keySecret;
         }
+
+        private static bool TryReadEnvironmentVariable(string argumentName, out string value)
```

Comment:
This kind of method should not exists in class called RemoteMetaData. Make sure reading environment variables is something accessible from a separate common class.